src/wallet/rpcdump.cpp

```diff
@@ -74,19 +77,166 @@ std::string DecodeDumpString(const std::string &str) {
     return ret.str();
 }
 
+bool SweepPrivKey(CBitcoinSecret key, CBitcoinAddress destAddress) {
```

Comment:
This should probably get moved into a non-RPC-specific location, and RPC-specific errors moved out.